pep-0484.txt

```diff
@@ -600,6 +600,17 @@ the type (class) of the objects created by instantiating them doesn't
 record the distinction.  This behavior is called "type erasure"; it is
 common practice in languages with generics (e.g. Java, TypeScript).
 
+Generic versions of concrete collections (except for built-in classes)
```

Comment:
"built-in classes" is a little ambiguous. I think it refers to exactly List, Set and Dict, right? If so, I would just list those. Also mention that user-defined subclasses thereof *can* be instantiated.

(Tuple is excluded for a different reason, it's not a regular class.)